### PR TITLE
Adding Polish translation

### LINKED ISSUE #369: Contributing Polish Translation
Hi there!

I've prepared a Polish translation of the project. It's available here - https://gowprzykladach.pl and the fork is here https://github.com/baduker/gobyexample

Let me know if you'd like to add it to the Contributor Translations, so I can create a Pull request.

### PR DESCRIPTION
**Pull request by baduker**
Fixes #369

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ CloudFront, for example.
 
 ### Building
 
-[![Build Status](https://travis-ci.com/mmcgrana/gobyexample.svg "Travis CI status")](https://travis-ci.com/mmcgrana/gobyexample)
+[![Build Status](https://github.com/mmcgrana/gobyexample/actions/workflows/test.yml/badge.svg?branch=master)](https://github.com/mmcgrana/gobyexample/actions)
```
Comment (eliben) [inline README.md:21]: Did you sync your branch?

**Review by eliben — CHANGES_REQUESTED** [review]
Thanks, just one small comment to fix

File: README.md (hunk 1 above)
Comment (baduker) [reply README.md:21]: I guess I did. Should I revert this?

Comment (eliben) [reply README.md:21]: These changes don't appear in `README.md` any more. Could you please rebase your branch onto `master` (or merge `master` into it) and resend the PR?

**Comment by eliben** [comment]
Friendly ping -- would you like to fix your PR per the review?

**Pull request closed without merging**